backend/src/test/java/gov/cdc/usds/simplereport/service/dataloader/PatientPrimaryPhoneDataLoaderTest.java

```diff
@@ -0,0 +1,71 @@
+package gov.cdc.usds.simplereport.service.dataloader;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.Person;
+import gov.cdc.usds.simplereport.db.model.PhoneNumber;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PhoneType;
+import gov.cdc.usds.simplereport.db.repository.PhoneNumberRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class PatientPrimaryPhoneDataLoaderTest {
+  int limit = 5000;
+  @Mock PhoneNumberRepository mockPhoneNumberRepository;
+
+  @Test
+  void dataloaderMakesOneCallToRepository_whenLessThanLimitItemsAreRequested() {
+    AtomicBoolean success = new AtomicBoolean();
```

Comment:
_googles AtomicBoolean_

neat! Actually, there are a few new-to-me Java things in these tests. I've got some reading to do 🕵️

Reply:
Same here Nathan 😂